packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {ConsoleLog} from '@bubblewrap/core';
+import {join} from 'path';
+import {existsSync, promises as fsPromises} from 'fs';
+import {loadOrCreateConfig} from '../config';
+
+async function jdkDoctor(log: ConsoleLog): Promise<boolean> {
+  const config = loadOrCreateConfig();
+  const jdkPath = (await config).jdkPath;
+  // Checks if the given path is a real path.
+  if (!existsSync(jdkPath)) {
+    log.error('The jdkPath doesn\'t exist, please run the following command to update it:\n' +
+    'bubblewrap updateConfig --jdkPath path/here. after thet run bubblewrap doctor again');
```

Comment:
Is `path/here` a placeholder for the path? It's usually better to use something between brackets when its a placeholder.
```suggestion
    'bubblewrap updateConfig --jdkPath <path-to-jdk>. After that, run bubblewrap doctor again.');
```